### DISCUSSION TITLE
Error playing MP3 files from SPIFFS '--- CUT HERE FOR EXCEPTION DECODER --'

### DISCUSSION THREAD

**Discussion by k-str50** [earlephilhower/ESP8266Audio#637] [General]
Hello,
I hope I'm not back in one of the widespread forums of the **_**category cemetery lost request**_**s.
For about 2 weeks I have been working on playing an mp3 file stored in the SPIFFS on a DAC. The file is available in the SPIFFS and the file size is also displayed correctly `file size =465189 `. When playing the file mp3->loop(), the process aborts after 9757 loops with the error message
`--------------- CUT HERE FOR EXCEPTION DECODER ---------------

Exception (3):
epc1=0x40204b04 epc2=0x00000000 epc3=0x00000000 excvaddr=0x4025a706 depc=0x00000000

>>>stack>>>

ctx: cont
sp: 3ffef730 end: 3ffefc00 offset: 0150
3ffef880:  000000b9 00000000 07520010 009b00d8  `
I would be very grateful for a helpful answer

**Comment by artificiel** [comment]
can you share the mp3, and also info on your hardware?

also you can squeeze more info from your stacktrace; depending on your toolchain/IDE you can pass the data to a tool that will point to lines in your code. for ESP32 it's called  `EspStackTraceDecoder.jar` which relies on `xtensa-esp32-elf-addr2line`.
